README.md

```diff
@@ -10,27 +10,32 @@ Code coverage:
 A utility that allows you to copy a Github issue to Jira
 
 ## Getting Started
-The gh2jira utility expects the following environment variables to be set: `GITHUB_TOKEN` and `JIRA_TOKEN`.
+The gh2jira utility requires a `config.yaml` file with Github Token, Jira Personal Access Token and Jira URL.Example `config.yaml` file:
 
-### Setting Up Github Token
-1. Follow the instructions [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) to create a personal access token, being sure to only limit the scope of the token to "public_repo" and "read:project".
-2. Copy the token created above and use it to set the `GITHUB_TOKEN` environment variable:
-```
-export GITHUB_TOKEN=<Copied Token>
+```yaml
+schema: gh2jira.config
```

Comment:
I really love the idea of introducing a config schema for this because it gives us the ability to iterate/evolve it and to accept/reject the info from basically the very first line.  

You will have to adjust the suite tests to include it though, or they'll fail.

Reply:
+1